Talk Proposals/Stop Worrying.md

```diff
@@ -0,0 +1,40 @@
+# How I Learned to Stop Worrying and Love Development
+
+## Abstract
+
+Is your life as a developer characterized by worry? Mine is. But we don't have to be stuck there, because worry can be a pointer to what we need to change. At this talk we'll discuss what you can do about your stress over schedule slips, defects, wasted testing effort, or building the wrong thing. You'll learn to apply agile development practices that will help turn your fear into confidence.
+
+## Description
+
+New programming languages and frameworks are great, but they don't address the fundamental problems we tend to experience on every project: long hours, late releases, and unhappy customers. We're told that agile development practices help with these problems, but getting started can be a catch-22. For example, agile suggests you should test your code, but to do so it has to be well-designed, but to change the design it needs to be tested. How do you get on the ride?
+
+I'd like to tell you the story of how I got on the ride — how I made it through a series of worries one at a time. Each worry came from a real need, and identifying that need helped me find an agile development process to meet it. Another worry would crop up afterward, but I just repeated the process of following the worry again. Now my day-to-day development involves a lot less stress and a lot more fun!
+
+Here's the series of worries I went through:
+
+- **"Will we get the code done in time?"** (The need here is for predictability.)
+- **"Will I break something?"** (The need is reliability.)
+- **"Am I wasting time writing tests?"** (Testability.)
+- **"Can my code handle future changes?"** (Flexibility.)
+
+For each of these worries, you'll learn:
+
+- Why the need underlying the worry is important to meet.
+- Which agile development practice addresses that need.
+- What that development practice looks like in a simple scenario.
+- Where you can go to dig more deeply into learning that development practice.
```

Comment:
Yeah, this all feels very abstract to me. It sounds kind of good, but I'm not sure what I'll be learning.